routes/users.js

```diff
@@ -1,15 +1,18 @@
 const express = require("express");
 const router = express.Router();
 const { create, findAll, findOne, update, destroy } = require("../controllers/user.controller.js");
+const { findOneValidation, createValidation, putValidation, destroyValidation } = require("../middleware/users-requests-validations.js");
+const { validateRequest } = require("../middleware/request-validation.js");
+
 router
   .route("/:id")
-  .get(findOne)
-  .put(update)
-  .delete(destroy);
+  .get(...findOneValidation, validateRequest, findOne)
```

Comment:
... is mandatory there?

Reply:
I am checking if the provided id is an valid id, should I delete it?